perftests/01-spiped-throughput.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/sh
+
+### Constants
+total=$( echo "2^24" | bc )
```

Comment:
I'm going to tweak these so that the user specifies which type of system, and it sets a sensible value accordingly.